examples/warehouse/service/dns-resolver-example-com/unbound.yaml

```diff
@@ -0,0 +1,15 @@
+# The unbound DNS resolver <https://unbound.net/> has a simple
+# key-value configuration structure with top-level categories and
+# underlying key-value items.  The YAML structure mirrors this.
+#
+# Value lists are represented as repeted key-values.  Strings in
```

Comment:
"repeated"

Also, what does this mean in practice? I see nothing in the examples resembling a "repeated key-value".